src/components/Add.js

```diff
@@ -0,0 +1,39 @@
+import React,{useState} from 'react';
+
+export default function Add(props) {
+    // Adding a hook to get the input and stores it.
+    const [input, setInput] = useState('');
+    
+    // Function that add items to the list
+    const addItem = () => {
+        if (!input) return null;
+        props.setItems((prevItems) => {
+            return [
+                ...prevItems, 
+                {
+                    id: Math.floor(Math.random() * 10000),
+                    item: input,
+                    completed: false,
+                    edit: false
```

Comment:
Convention for boolean variables is to answer a question. `isCompleted` `isEditing`.